guides/common/modules/proc_assigning-content-view-environments-to-an-activation-key-by-ID.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Assigning_content_view_environments_to_an_activation_key_by_ID{context}"]
+= Assigning content view environments to an activation key by ID
+
+To assign content view environments to an activation key, determine the IDs of the content view environments you want to assign.
+
+.CLI procedure
+
+. Run the `hammer content-view-environment list` command and note the IDs you want to use.
+. Run the following command to assign the selected content view environments to an activation key by passing their IDs as an array:
++
+[options="nowrap" subs="+quotes"]
+----
+hammer activation-key update --organization-id
```

Comment:
the command is missing here. Should be something like
```
hammer activation-key update --organization "My_Organization" --content-view-environments "My_env1/My_cv1,My_env2/My_cv2"
```

Reply:
Command prompt is also missing. See [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#commands-in-code-blocks).

Vale did not flag this because the rule was designed for repos that use `[source,terminal]` for code blocks. I will follow this up.